server/backends/blobstore/blobstore_test.go

```diff
@@ -0,0 +1,61 @@
+package blobstore
+
+import (
+	"context"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestDiskBlobStore(t *testing.T) {
+	for _, tc := range []struct {
+		name             string
+		prefix, blobName string
+		blob             []byte
+	}{
+		{
+			"without_prefix",
+			"",
+			"test_blob",
+			[]byte("test"),
+		},
+		{
+			"with_prefix",
```

Comment:
uber nit: since the `name` becomes part of the test name, it looks slightly better IMO to have it in CapitalCase

TestDiskBlobStore/WithoutPrefix: PASS
TestDiskBlobStore/WithPrefix: PASS

```suggestion
			"WithoutPrefix",
			"",
			"test_blob",
			[]byte("test"),
		},
		{
			"WithPrefix",
```